components/src/dynamo/vllm/args.py

```diff
@@ -454,7 +454,7 @@ def overwrite_args(config):
         # skip tokenizer initialisation.  Setting this to **False** avoids
         # a NoneType error when the processor accesses the tokenizer.
         "skip_tokenizer_init": False,
-        "disable_log_requests": True,
```

Comment:
I think enable_log_requests is inherently false by default, so if we just remove this line I think it will work as expected. The important point here is, I think if we remove this line, users can still opt-in by explicitly providing `python -m dynamo.vllm ... --enable-log-requests`, but if you keep this line here, I think we will always disable it without allowing user override from CLI. Please correct me if wrong.

CC @alec-flowers

Reply:
I've tried to preserve the previous behaviour with these changes, so I've mechanically replaced the deprecated flag and its assigned values with the new one and the corresponding values.

But I agree that allowing to override this from the CLI would be nice.